MekHQ/src/mekhq/campaign/personnel/familyTree/Genealogy.java

```diff
@@ -223,7 +217,7 @@ public boolean hasParents() {
      * @return true if they have mutual ancestors, otherwise false
      */
     public boolean checkMutualAncestors(Person person, Campaign campaign) {
-        if (getOrigin().equals(person.getId())) {
+        if (getOrigin().equals(person)) {
```

Comment:
Is getOrigin() guaranteed to be non-null?

Reply:
It wasn't only for a few error conditions, but that will be fixed... swapped around a few XML bits to improve and fix that potential issue.